knowledge-base/datagrid-cell-text-highlight.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Highlighting Text in the DataGrid through search Entry
+description: Highlight Text in DataGrid cell through an Entry
+type: how-to
+page_title: Cell Text Highlight through a search
+slug: datagrid-cell-text-highlight
+position: 
+tags: MAUI
+ticketid: 1582978
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>3.2.1</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>DataGrid for MAUI</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+How can I have highlighted matching text from a search entry inside Telerik UI for .NET MAUI DataGrid Cell?
```

Comment:
In this article I will show you how to highlight a text inside a DataGrid cell when searching from an external UI